### DISCUSSION TITLE
Town spawn warmup not working

### DISCUSSION THREAD

**Discussion by g5bb** [TownyAdvanced/Towny#5684] [Q&A] [answered]
I'm not sure why, but the warmup of teleporting to town spawns is not working. When teleporting to a town's spawn, it instantly teleports instead of waiting 3 seconds.  The cooldown of teleportation does work though. This is my configuration below, I have checked my permissions and it doesn't seem I have any permission to bypass the warmup.

`teleport_warmup_time: '3'`

This is what it's set to in my config. Anyone know a fix to this problem?

Edit: Nevermind, it fixed itself. I'll just mark this as answered.

**Comment by LlmDl** [comment, marked as answer]
This is when you should be filing a bug report on the Issues tracker.
